packages/terra-framework-docs/CHANGELOG.md

```diff
@@ -26,6 +26,9 @@
   * Added documentation and tests for `terra-worklist-data-grid`.
   * Added example and test for masked cell feature for `terra-worklist-data-grid`.
 
+* Added
+  * Added example and test for masked cell feature for `terra-worklist-data-grid`.
+
```

Comment:
Does this need to be under the `unreleased` section?

Reply:
i'll move it.